tests/integration/src/test/scala/mesosphere/IntegrationTest.scala

```diff
@@ -0,0 +1,72 @@
+package mesosphere
```

Comment:
Are we planning to move all other tests as well? Shouldn't the path be `tests/integration/src/test/scala`?

Reply:
Lol, I misread that one ^^.